practice/lib/practice/mid_char.ex

```diff
@@ -15,7 +16,7 @@ defmodule MidChar do
     |> Kernel.-(1)
     |> div(2)
   end
-  
+
   def pull_num_of_elem(count) when rem(count, 2) == 0, do: 2
   def pull_num_of_elem(_), do: 1
```

Comment:
I've started gravitating away from very small functions like this, but other people will probably argue for them. If you in-line them you could probably create a more clear pattern of data transformation by piping more inside the `get_middle` function. Caremeter 3/10

Reply:
There's a very interesting paragraph or three in McConnell's [_Code_Complete_](http://www.worldcat.org/oclc/1073686696) about the subject of operations that seem to simple to put into subroutines (around page 166 of the Second Edition).  Highly recommended reading regardless but it is quite germane to your point. 

I won't rehash his excellent prose but in essence he argues that it's often better to put things in small, single-purpose subroutines even if it seems like they may be overengineering.

Reply:
@apreifsteck I've seen both sides of the argument. Most of my dev friends that work externally like it but when I posted it to twitter, the feedback I got lol.
@OnorioCatenacci I'll have to check the book out. I personally enjoy these  one-line functions mostly because I like breaking code into the smallest piece possible.